editor/src/clj/editor/pipeline/fontc.clj

```diff
@@ -592,13 +592,14 @@
                                 edge
                                 shadow-blur
                                 shadow-alpha]
-  (let [^int padding padding
-        ^double outline-width outline-width
-        ^double sdf-spread sdf-spread
-        ^double sdf-shadow-spread sdf-shadow-spread
-        ^double edge edge
-        ^double shadow-alpha shadow-alpha
-        ^double sdf-outline sdf-outline
+  (let [glyph-left-bearing (double glyph-left-bearing)
+        ^int padding padding
+        outline-width (double outline-width)
+        sdf-spread (double sdf-spread)
+        sdf-shadow-spread (double sdf-shadow-spread)
+        edge (double edge)
+        shadow-alpha (double shadow-alpha)
+        sdf-outline (double sdf-outline)
```

Comment:
Hinting locals as `^double` is not enough apparently: locals keep being objects that get cast to double on use. This place generated a lot of `Double`s. How I figured this out: I reloaded the whole ns, and then did `(compile 'editor.pipeline.fontc)`. This dumped the classes on disc. Then, I opened `target/classes/editor/pipeline/fontc$draw_ttf_distance_field.class` and saw a lot of `RT.uncheckedDoubleCast`s and `(Number)a_local`s.

Reply:
Whaaat? 😨